configure.ac

```diff
@@ -1383,6 +1393,35 @@ AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
     AC_MSG_RESULT(no)
 ])
 
+have_libxdp=no
```

Comment:
This logic is responsible for creating HAVE_LIBXDP macro. In case the compilation of the given code is successful then HAVE_LIBXDP is defined

Reply:
I dont worry about this much but having libxdp != AF_XDP support.
For now its fine, however somehow we should test if AF_XDP socket is available.

Probably a simple socket(AF_XDP, SOCK_RAW, 0) dummy call's return value can verify that.

Reply:
I extended the current logic with your suggestion however at this point we only have compilation check, due to the fact that we are using LIBXDP it is important to check whether a utility function from the library can be compiled. I think socket(AF_XDP, SOCK_RAW, 0) would compile in cases when there is no AF_XDP support